File: Upcoder/Lập trình cơ bản/2.Lệnh rẽ nhánh/In ấn/INAN.py
# Nhập dữ liệu từ bàn phím
input_data = input().split()
T = int(input_data[0])
P = int(input_data[1])
C = int(input_data[2])

if T == 1:
    A4 = P * C
    print(A4)
elif T == 2:
    if P % 2 == 0:
        A4 = (P // 2) * C
    else:
        A4 = ((P + 1) // 2) * C
    print(A4)

File: Upcoder/Lập trình cơ bản/2.Lệnh rẽ nhánh/In ấn/INAN.cs
using System;

namespace Inan
{
    public class Program
    {
        static void Main(string[] args)
        {
            string [] input = Console.ReadLine().Split(' ');
            long T, P, C, A4;
            T = long.Parse(input[0]);
            P = long.Parse(input[1]);
            C = long.Parse(input[2]);
            if (T == 1)
            {
                A4 = P * C;
                Console.WriteLine(A4);
            }
            else if (T == 2)
            {
                if (P % 2 == 0)
                {
                    A4 = (P / 2) * C;
                }
                else
                {
                    A4 = ((P + 1) / 2) * C;
                }
                Console.WriteLine(A4);
            }
        }
    }
}
